themes/theme-gmd/pages/Login/index.jsx

```diff
@@ -272,6 +272,7 @@ class Login extends Component {
                         ? i18n.text('login.hide_password')
                         : i18n.text('login.show_password')
                       }
+                      className={styles.toggleButton}
```

Comment:
as discussed the icons need a rework. 

- please create a new `PersonIcon.jsx` inside `libraries/ui-shared/icons`.
- you can use `libraries/ui-shared/icons/HeartPlusOutlineIcon.jsx` as blueprint. 
- add the svg path from the `extension-config.json` as value for the `fallback` constant.
- also add an export for the new icon to `libraries/engage/components/index.js`.
- set the value for the person icons inside the extension-configs to `""`

rework the `Login` pages to use the named icon components. import everything from `@shopgate/engage/components`.